doc/tutorial/unix/tutorial-install-ubuntu.doc

```diff
@@ -31,7 +31,7 @@ In this section, we give minimal instructions to build ViSP from source just to
 
 - Install a small number of recommended 3rd parties
 \code
-$ sudo apt-get install libopencv-dev libx11-dev liblapack-dev libeigen3-dev libdc1394-22-dev libv4l-dev libzbar-dev libpthread-stubs0-dev libjpeg-dev libpng-dev
+$ sudo apt-get install libopencv-dev libx11-dev liblapack-dev libeigen3-dev libdc1394-dev libv4l-dev libzbar-dev libpthread-stubs0-dev libjpeg-dev libpng-dev
```

Comment:
`libdc1394-dev` is available only from `20.04`, see: https://packages.ubuntu.com/search?keywords=libdc1394-dev&searchon=names&suite=all&section=all

A note should be added.

Reply:
You are right. Thanks